client/src/app/components/client-gridview/client-gridview.component.ts

```diff
@@ -55,4 +56,16 @@ export class ClientGridviewComponent implements OnInit {
       this.changeDetectorRefs.detectChanges();
   }
 
+  updateList(id: number, property: string, event: any): void {
+    const editField = event.target.textContent;
+    this.users[id][property] = editField;
+    this.manageMembersService.modifyClientInfo(property, editField, id).subscribe((user) => {
+      const users = user;
+      this.populateTable(users);
+    });
```

Comment:
You should handle errors that are sent back by the API. Check out how it's done in other components.